lib/linter/linter.js

```diff
@@ -1248,7 +1248,8 @@ class Linter {
             options.filterCodeBlock ||
             (blockFilename => blockFilename.endsWith(".js"));
         const originalExtname = path.extname(filename);
-        const messageLists = preprocess(text, filenameToExpose).map((block, i) => {
+        const processorOptions = config.processorOptions || {};
```

Comment:
This `|| {}` was a late addition, and something I'm not too sure about. When `config` is constructed via config extraction, it will default to containing `processorOptions: {}`, but if someone just manually calls `Linter#verify`, it will not. It seems it would be nice if processors could always assume that they're getting an object. Is there a better way or another place this should happen?

Reply:
This looks good to me.